packages/web-components/src/components/va-link-action/va-link-action.css

```diff
@@ -85,4 +85,84 @@
   vertical-align: top;
   max-width: calc(100% - 24px);
   white-space: pre-wrap;
+}
+
+:host(.host--primary-entry) {
+  width: 100%;
+}
+
+:host a.va-link--primary-entry {
+  position: relative;
+  display: inline-block;
+  background-color: var(--vads-color-primary);
+  color: var(--vads-color-white);
+  text-decoration: none;
+  padding: 15px 15px 15px 45px;
+  margin: 0 1.5rem 0 0;
+  clip-path: polygon(0 0, calc(100% - 15px) 0, 100% 50%, calc(100% - 15px) 100%, 0 100%);
+  transition: all 0.2s ease-in-out;
```

Comment:
a11y should confirm but I think this transition easing is not an ideal behavior when focused. It's challenging on the eyes to see the easing delay going from blue to yellow.

Adding `transition: none` to the focus styles might take care of it.

https://github.com/user-attachments/assets/e407ff01-d028-43a0-97cb-f2a18fc4ec0d

Reply:
Fair enough. Transition set to none.

Reply:
There is still a delay with the icon so the text and icon aren't in sync. That's also different from the other variation's too. Is it possible to remove the transition for the icon as well?


https://github.com/user-attachments/assets/357031a9-58dd-4473-9f34-9b2f46d96e6b